packages/lake/src/components/SegmentedControl.tsx

```diff
@@ -1,26 +1,55 @@
-import { ReactNode, useRef, useState } from "react";
+import { ReactNode } from "react";
 import { StyleSheet, View } from "react-native";
-import { commonStyles } from "../constants/commonStyles";
-import { backgroundColor, colors, radii, spacings } from "../constants/design";
+import { backgroundColor, colors, radii, spacings, texts } from "../constants/design";
 import { isNotNullish } from "../utils/nullish";
 import { Box } from "./Box";
-import { Icon } from "./Icon";
 import { LakeText } from "./LakeText";
 import { Pressable } from "./Pressable";
 import { Space } from "./Space";
 
-const HEIGHT = 64;
-
-const styles = StyleSheet.create({});
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.gray[50],
+    borderRadius: radii[8],
+    padding: spacings[4],
+  },
+  selectedItem: {
```

Comment:
That's not really `selectedItem`, more `selectedItemIndicator` (it's the animated white square)